Cargo.toml

```diff
@@ -70,3 +75,19 @@ facebook          = ["hyper/ssl", "openssl"]
 portaudio-backend = ["portaudio"]
 pulseaudio-backend= ["libpulse-sys"]
 default           = ["with-syntex", "portaudio-backend"]
+
+[package.metadata.deb]
+maintainer = "nobody"
+copyright = "2015 Paul Lietar"
```

Comment:
That should be 2016.
Also a small detail, but could you put the accent on my name please 😄